src/components/Import.js

```diff
@@ -10,21 +10,24 @@ import {
   Menu,
   MenuItem,
 } from "@material-ui/core";
-// import { MoreVert } from "@material-ui/icons";
 import IconButton from "@material-ui/core/IconButton";
-import MoreVertIcon from '@material-ui/icons/MoreVert';
+import MoreVertIcon from "@material-ui/icons/MoreVert";
 
 const Import = (props) => {
   // fill out this component
   const [anchorEl, setAnchorEl] = React.useState(null);
-  const open = Boolean(anchorEl);
+  const [currentIndex, setCurrentIndex] = React.useState(-1);
```

Comment:
@hbrashid This is how you use state in a functional component, but you could switch this to a class component to use state the way we've learned so far.

Reply:
I see. And the '-1' is the current state. Why is it '-1'?